### ISSUE TITLE
Getting choice probabilities from fitted Large MNL models

### ISSUE THREAD

**Issue by smmaurer** [UDST/urbansim_templates#31]
Brainstorming with @janowicz about functionality for the Large MNL template:

After fitting a model, it's useful to be able to get choice probabilities for the full list of alternatives, for diagnostics. The probabilities could be for a single representative agent or for multiple. This is an inexpensive calculation because it's just multiplication.

(Currently, the probabilities are saved to the class object when the template is "run". But this doesn't satisfy the diagnostic use case, where we might want to use different data tables and skip the simulated choices.)

### Implementation

I think it makes sense for this to be ChoiceModels functionality that's easy to access using intermediate outputs from the template. (A user might want to use different filters or sampling for these diagnostics than in the model step itself.)

The syntax could be something like this:

```py
results = m.get_fitted_model()  # type = choicemodels.MultinomialLogitResults()
df = choicemodels.tools.MergedChoiceTable(m.get_choosers().loc[0], m.get_alternatives())
probs = results.probabilities(df)

# or if the model expression only includes attributes of the alternatives:
probs = m.get_fitted_model().probabilities(m.get_alternatives())
```

We'll need to implement `m.get_choosers()`, `m.get_alternatives()`, `m.get_fitted_model()`, and a fuller-featured results class for `choicemodels.MultinomialLogit`, including a `probabilities()` method.

The latter would be part of the general extension and refactoring of MNL prediction tools that's in the works; see [ChoiceModels issue #26](https://github.com/UDST/choicemodels/issues/26).

**Comment by smmaurer** [comment]
Update: [ChoiceModels PR #41](https://github.com/UDST/choicemodels/pull/41) provides functionality for getting the probabilities. Still need to adjust the template to work more smoothly with this.

**Comment by smmaurer** [comment]
Resolved with PR #79
